resources/zh/cemu.po

```diff
@@ -569,49 +610,42 @@ msgid "Forces a given controller profile"
 msgstr "强制给定控制器设定档"
 
 #: GameUpdateWindow.cpp
-msgid "Can't open the file."
-msgstr "无法打开文件."
-
-#: GameUpdateWindow.cpp
-msgid "Invalid \"meta.xml\" file."
-msgstr "无效的\"meta.xml\"文件."
-
-#: GameUpdateWindow.cpp
-msgid "Can't find the \"title_id\" in the given \"meta.xml\" file."
-msgstr "无法在选择的\"meta.xml\"文件中找到“title_id”."
+msgid ""
+"It seems that there's a wrong title installed at the target location.\n"
```

Comment:
Under normal circumstances, the similar function in the Chinese context is called the game manager, although it also manages dlc and updates. In view of the fact that "title" has appeared many times in this software, and it is different from "game" when used, it cannot be translated into a game manager here.
I realized this, which is why I translated it into 档案. File means a single file in Chinese, and 档案 usually means a set of archived and valuable files. Translation into 档案 is the result of my careful consideration. This is a more suitable translation that I can currently consider.

Reply:
Understood. But considering a "title" as a "a set of archived and valuable files" is a bit too generic for this scenario.

Reply:
In a sense, "档案" is indeed broader. It can mean "publication", but of course it is not accurate enough. The "档案" in the vocabulary that can be found at present should be a relatively localized translation, and it can also allow users to understand the function of this title manager. If there is a better translation later, it will definitely be changed.